edgehog/device/forwarder/http.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package edgehog.device.forwarder;
+
+message Http {
+    oneof message {
+        Request request = 1;
+        Response response = 2;
+    }
+
+    message Request {
+        bytes request_id = 1;
+        string path = 2;
+        string method = 3;
+        optional string querystring = 4;
```

Comment:
All fields are optional by default, why do we need explicit `optional` here? Google's [API design guide](https://cloud.google.com/apis/design/proto3) says that using `optional` in Protobuf 3 is "generally discouraged"

Reply:
Also, imho `query_string` is better since they're two separate words

Reply:
Ok, done.

Reply:
I’d just like to interject for a moment: Protobuf >= 3.14 does not make any distinction between missing data or zero-valued data (e.g. `""` for strings, `false` for booleans) , see https://protobuf.dev/programming-guides/encoding/#optional.
Therefore, if there is a significant difference between a missing field and a present field whose value is the same as the default one, you should leave the field as `optional`.

Reply:
After some thought, I think that no fields need to be optional since we can check for an empty value in the application code.